### ISSUE TITLE
Questions about VTAB-1K CIFAR100

### ISSUE THREAD

**Issue by WangYZ1608** [dongzelian/SSF#20]
Hello, congratulations!
Thank you for released VTAB-1K, it help me a lot.
I tried to experiment on CIFAR100, but found that the results of SSF and VPT were far worse than those in the paper.
I suspect there is a problem with the labels of the released CIFAR-100 dataset. So I wanted to ask you to check it out.
grateful.

**Comment by 2catycm** [comment]
I got a 67.47 acc1

**Comment by 2catycm** [comment]
what about you?

**Comment by kennethyangg** [comment]
@2catycm @WangYZ1608 Hi, I'm wondering where do you guys find the train800val200.txt and test.txt provided in codes?

**Comment by 2catycm** [comment]
actually, the trainval 800 i think is bad,its classes are not stratified. some classes in val are not presented in train. I did a re train val split by sklearn



---Original---
From: "Kenneth ***@***.***&gt;
Date: Fri, Oct 18, 2024 12:18 PM
To: ***@***.***&gt;;
Cc: "Ye Canming ***@***.******@***.***&gt;;
Subject: Re: [dongzelian/SSF] Questions about VTAB-1K CIFAR100 (Issue #20)




 
@2catycm @WangYZ1608 Hi, I'm wondering where do you guys find the train800val200.txt and test.txt provided in codes?

**Comment by kennethyangg** [comment]
@2catycm Thank you for your reply! I'm curious whether the author provides the dataset splits or if it's something we need to create ourselves?
